src/main.rs

```diff
@@ -25,6 +26,15 @@ async fn main() -> Result<()> {
     let api = Api::new();
     let db = Database::new().await?;
 
+    let val = api
```

Comment:
Do we want to check this into main or was it just for debugging?

Reply:
removed